src/Compiler/Service/SemanticClassification.fs

```diff
@@ -58,7 +58,6 @@ type SemanticClassificationType =
     | TypeDef = 35
     | Plaintext = 36
 
-[<RequireQualifiedAccess>]
```

Comment:
I don't think it's right fix. We should instead bump used fslib package version
https://github.com/dotnet/fsharp/blob/63ff188355f0e916d75aced242f02ccf0f11626b/eng/Versions.props#L42

We can either set it to released `8.0.300` or use previews: https://dev.azure.com/dnceng/public/_artifacts/feed/dotnet8/NuGet/FSharp.Compiler.Service/versions/43.8.400-preview.24267.3

Reply:
I can do both, the attribute usage does not have effect (it is incorrect, so the error/warning is valid).